src/design-system/asset-input/amount-input.component.tsx

```diff
@@ -10,14 +10,17 @@ interface Props {
   onChange?: (value: string) => void;
   value?: string;
   id: string;
+  'data-testid'?: string;
```

Comment:
I believe the pattern we follow for data-testid is:
- component's prop name: `testId`
- prop applied to element `data-testid={testId}`

Reply:
done in latest push